### PR TITLE
Translate Fonts 2020 Chapter to Dutch + typo's

### PR DESCRIPTION
**Pull request by noah-vdv**
#1750 
Also, line 341 `font-weight: [number]`  `font-stretch: [number]%` is the `%` a typo?

### CODE REVIEW

File: src/content/nl/2020/fonts.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+  sheets_gid="309969915",
+  sql_file="04_17.VF_axis_value.sql"
+  )
+}}
+
+Verreweg de meest gebruikte as is `wght` (die het gewicht regelt), met 84,7% desktop en 90,4% mobiel. Echter, `wdth` (breedte) is verantwoordelijk voor ongeveer 5% van het gebruik van variabele fonts. In 2020 begon Google Fonts met het aanbieden van 2-assige lettertypen met zowel breedte- als gewichtsassen.
+
+Het is vermeldenswaard dat de voorkeursmethode het gebruik van `font-weight` en `font-stretch` is in plaats van de syntaxis van `font-variation-settings` op een lager niveau voor deze twee assen, aangezien ze volledig worden ondersteund door alle browsers die ondersteuning bieden voor variabele lettertypen. Door het gewicht in te stellen via `font-weight: [number]` en breedte via `font-stretch: [number]%`, geven auteurs meer geschikte stijlhints aan de browser, wat op zijn beurt een betere weergave mogelijk maakt voor de eindgebruiker als de variabele lettertype kan niet worden geladen. Dit voorkomt ook dat de normale overerving van stijlen via de cascade wordt gewijzigd.
```
Comment (noah-vdv) [inline src/content/nl/2020/fonts.md:341]:
```suggestion
Het is vermeldenswaard dat de voorkeursmethode het gebruik van `font-weight` en `font-stretch` is in plaats van de syntaxis van `font-variation-settings` op een lager niveau voor deze twee assen, aangezien ze volledig worden ondersteund door alle browsers die ondersteuning bieden voor variabele lettertypen. Door het gewicht in te stellen via `font-weight: [number]` en breedte via `font-stretch: [number]`, geven auteurs meer geschikte stijlhints aan de browser, wat op zijn beurt een betere weergave mogelijk maakt voor de eindgebruiker als de variabele lettertype kan niet worden geladen. Dit voorkomt ook dat de normale overerving van stijlen via de cascade wordt gewijzigd.
```
This^

**Comment by bazzadp** [comment]
> Also, line 341 `font-weight: [number]` `font-stretch: [number]%` is the `%` a typo?

Interesting question! I think the fact it's a number, means it's fine the percentage symbol is there as numbers need to be percentages. However it is possible to give non-numeric values [as shown in the MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/font-stretch), but given the text explicitly says `[number]` it should be a percentage.

So, in summary, I think it's fine the percentage is there.

File: src/content/nl/2020/fonts.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+chapter_number: 4
+title: Lettertypen
+description: Hoofdstuk over lettertypen van de Web Almanac 2020 waarin wordt beschreven waar lettertypen worden geladen, lettertype-indelingen, laadprestaties voor lettertypen, variabele lettertypen en kleurlettertypen.
+authors: [raphlinus, jpamental]
+reviewers: [RoelN, svgeesus, davelab6, rsheeter, mandymichael]
+analysts: [AbbyTsai]
+editors: [bazzadp]
+translators: [noah-vdv]
+jpamental_bio: Ontwerper, knutselaar, typograaf. Auteur van Responsive Typography, uitgenodigde expert voor het W3C en 10+ jaar ervaring gericht op betere typografie op internet.
+raphlinus_bio: Raph Levien werkt al meer dan 35 jaar met lettertypen, waaronder een doctoraat aan UC Berkeley in hulpmidelen voor het ontwerpen van lettertypen. Hij voegt zich weer bij <a href="https://fonts.google.com/">Google Fonts</a> als onderzoeker op het gebied van lettertypetechnologie, nadat hij het team in 2010 mede heeft opgericht.
```
Comment (bazzadp) [inline src/content/nl/2020/fonts.md:12]:
```suggestion
raphlinus_bio: Raph Levien werkt al meer dan 35 jaar met lettertypen, waaronder een doctoraat aan UC Berkeley in hulpmidelen voor het ontwerpen van lettertypen. Hij voegt zich weer bij <a hreflang="en" href="https://fonts.google.com/">Google Fonts</a> als onderzoeker op het gebied van lettertypetechnologie, nadat hij het team in 2010 mede heeft opgericht.
```

File: src/content/nl/2020/fonts.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=2012634758&format=interactive",
+  sql_file="04_05.web_font_usage_breakdown_with_fcp.sql"
+  )
+}}
+
+Bij het bekijken van commerciële services zoals Adobe (`use.typekit.net`) of Monotype (`fast.fonts.com`) is het interessant om op te merken dat ze op desktop net zo snel of iets sneller zijn dan Google Fonts, maar merkbaar langzamer op mobiel. De conventionele wijsheid heeft over het algemeen aangenomen dat de trackingscripts die door die services worden gebruikt, ze aanzienlijk vertragen, maar dat is blijkbaar tegenwoordig minder een probleem dan in de afgelopen jaren. Hoewel het waar is dat we de prestaties van de site meten en niet noodzakelijk de prestaties van de font-hoster, hebben deze trackingscripts invloed op het laden van lettertypen op de client, dus het lijkt relevant om deze observaties op te nemen.
```
Comment (bazzadp) [inline src/content/nl/2020/fonts.md:105]: Did you see I changed this in #1933? Wasn't happy with the word "font hoster" so changed this to:

> While it's true that we are measuring site performance and not necessarily performance of the font host, those tracking scripts impact font loading on the client so it seems relevant to include these observations.

Not sure if you want to make a similar change here?

**Review by bazzadp — COMMENTED** [review]
Looks good! And thanks for spotting (and fixing!) the English typos.

Made a couple of comments.

**Comment by bazzadp** [comment]
Can you also set Rod's proper name in 2020.json, if you don't mind, as part of this?

```json
    "rsheeter": {
      "name": "Rod Sheeter",
      "teams": [
        "reviewers"
      ],
      "github": "rsheeter"
    },
```

**Review by bazzadp — APPROVED** [review]
Thanks!!!

**Pull request merged**